src/index.html

```diff
@@ -20,8 +20,396 @@
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img-computer"
+          src="./images/imac.jpeg"
+          alt="product laptop"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <span class="card__product-code">Product code: 195434</span>
+
+        <div
+          class="card__wrap-stars-reviews"
+          data-qa="card"
+        >
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__wrap-price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          class="card__order-btn"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__img-computer"
```

Comment:
Fix classes everywhere
```suggestion
          class="card__img"
```